imports/plugins/core/ui-tagnav/client/containers/tagNavContainer.js

```diff
@@ -114,6 +116,7 @@ const wrapComponent = (Comp) => (
     componentDidMount() {
       window.addEventListener("resize", this.onWindowResize);
       this.onWindowResize();
+      this._isMounted = true;
```

Comment:
What is prefacing this variable with an underscore supposed to indicate/accomplish?

Reply:
I prefixed that variable with an underscore to show that it's a custom implementation of the now deprecated `ReactComponent.isMounted()` as advised in [this official React post](https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html).

Reply:
Furthermore, it's consistent with the `_unmounted` variable in `imports/plugins/core/components/lib/composer/compose.js`.

Reply:
Great. Works for me.